xbmc/windowing/android/AndroidUtils.cpp

```diff
@@ -123,6 +124,26 @@ static void fetchDisplayModes()
   }
 }
 
+namespace
+{
+  std::string HdrTypeString(int type)
```

Comment:
Wrong indentation

Reply:
Yes I'm applying Jenkins' fixes in the next push.